java-checks/src/test/files/checks/security/CryptographicKeySizeCheck/RSA.java

```diff
@@ -0,0 +1,74 @@
+import java.security.KeyPairGenerator;
+import java.security.NoSuchAlgorithmException;
+import javax.crypto.KeyGenerator;
+
+class RSA {
```

Comment:
I think that there are too many tests here which duplicate what's tested for Blowfish.

Reply:
You are right no need to keep everything.
I just left a Compliant and a Noncompliant Case for RSA.